configs/minimal.yaml

```diff
@@ -187,3 +186,14 @@ VIEW_FREEZE_DEADLINE: 3
 MAX_REQUEST_INCLUSION_LIST: 16
 # 2**13 (= 8192)
 MAX_BYTES_PER_INCLUSION_LIST: 8192
+
+# Blob Scheduling
+# ---------------------------------------------------------------
+
+BLOB_SCHEDULE:
+  # Deneb
+  - EPOCH: 18446744073709551615
+    MAX_BLOBS_PER_BLOCK: 6
+  # Electra
```

Comment:
Should we add a `MAX_BLOBS_PER_BLOCK` for fulu? 

Currently how this is drafted, `get_max_blobs_per_block()` will return 9 during fulu.

Reply:
Nah, during the 2nd to last PeerDAS breakout call we decided to remove any placeholder blob limit increases. We can decide these in a future PR. This PR just lays the foundation for that. Also, the current plan initially is to keep the Electra blob limit at the fork, then 2x two weeks later, then 2x two months later, and 2x once again another two months later. Resulting 72 blobs per block 4.5 months after Fulu.

Reply:
> then 2x two weeks later

how did we come up with that number btw? 2 weeks doesn't provide us any value, it's not sufficient to analyse the data on the network and push out new client releases + get everyone to update, minimum should be 2 months imo

Reply:
This comes from Alex Stokes's proposal: [blob/acc in 2025](https://hackmd.io/@ralexstokes/blob-acc-2025). There's a some rationales in there. To be clear, this is just his proposed schedule. Nothing has been decided yet. And this discussion is out-of-scope for this PR 😅

Reply:
I don't really see much rationale there but saw someone else left similar comment on the document, but agree this is out of scope of the PR :grin: